src/csvcubed/models/validationerror.py

```diff
@@ -26,51 +27,75 @@ class SpecificValidationError(ValidationError, HasErrorUrl, ABC):
 
 
 @dataclass
-class PydanticValidationError(ValidationError, ABC):
-    """
-    The error type returned from pydantic validation functionality.
-    """
+class ValidateModelPropertiesError(ValidationError):
 
-    path: List[str] = field(init=False, default_factory=lambda: [])
     """
-    Place to for pydantic validation to set the object path where this error occurred.
+    This error will be returned with the name of the variable that has not been validated.
```

Comment:
It's not the name anymore, it's the path to the offending value.

Reply:
done